ebpf/tracer_linux_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build linux
+// +build linux
```

Comment:
why do we need to have these 2 directives? won't just `//go:build linux` do the job? 🤔